tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -377,6 +381,7 @@ public static int setDateOrder(int dateOrder) {
       // ResultSet ---------------------------------------------------------
     $RESULT_SET_FETCH_ROWNUM("tajo.resultset.fetch.rownum", 200),
```

Comment:
@hyunsik 
I think it should be a size. If a row size is too big, probably it can be OOM